docs/genai/howto/build-from-source.md

```diff
@@ -87,9 +87,10 @@ Build from source and copy the include and libraries into `ORT_HOME`
 On Windows
 
 ```cmd
-build.bat --build_shared_lib --skip_tests --parallel [--use_cuda]
+build.bat --build_shared_lib --skip_tests --parallel [--use_cuda] --config Release
 copy include\onnxruntime\core\session\onnxruntime_c_api.h <ORT_HOME>\include
 copy build\Windows\Debug\Debug\*.dll <ORT_HOME>\lib
```

Comment:
same here

Reply:
maybe change Release to RelWithDebInfo to keep consistent with Linux

Reply:
So we can do RelWithDebInfo on Windows too?